Todo List Project/assets/js/todoList.js

```diff
@@ -1,7 +1,6 @@
 
 var textIn = document.getElementById("newLineInput");
```

Comment:
Make sure you understand the difference between `var`, `let`, and `const`

Reply:
Not really enough to use let and const. I know let will be local to a block of code, like a for loop. Constant is good when you won't have any changes.

Reply:
i had some a lot of bugs in my delete line function. it would always delete the last item on the list when i used "var nested = listItems[i];

 
function deleteLine() {
	let nested = listItems[i];
	trashBtn[i].addEventListener("click", function() {
		list.removeChild(nested);
		// stop event propigation
		event.stopPropagation();
	});
}

Reply:
`let` is not local to a block of code. It allows for reassignment where `const` does not. If you know that the variable will not change then it is better to use `const` for security and safety concerns. If you are unsure then it is best to use `let`.

Reply:
It looks to me like you are not passing your iterator into your `deleteLine` function. How does deleteLine know which iteration it is on? You call the function inside of the `for` loop but you do not pass the iterator `i` into the function.